libmscore/harmony.cpp

```diff
@@ -735,6 +735,14 @@ bool Harmony::edit(EditData& ed)
 
 void Harmony::endEdit(EditData& ed)
       {
+      _userName.replace("\u1d12b", "bb"); // double-flat
+      _userName.replace("\u226d",  "b");  // flat
+      _userName.replace("\u266f",  "#");  // sharp
+      _userName.replace("\u1d12a", "x");  // double-sharp
+      _userName.replace("\u0394",  "t");  // &Delta;
+      _userName.replace("\u00d0",  "o");  // &deg;
+      _userName.replace("\u00f8",  "0");  // &oslash;
+      _userName.replace("\u00d8",  "0");  // &Oslash;
       TextBase::endEdit(ed);
```

Comment:
in 2.2 this is `Text::endEdit();`, guess that might be why the above replacements are not working in master?

Reply:
No, it isn't that, stepping thru with a debugger shows that the replacement happens, but gets reverted by the `endEdit`

Reply:
Odd, it worked for me when I tried it.  But there is stuff going on with transposition and links that could be complicating this.  I'm not worried, this strikes me as extremely low priority.  The supported way of entering these symbols is easier.

Reply:
I agree. The discussion here shows that it will be hard to add this in 2.2 since we don't know what we will break :)

Reply:
Well, the problem is that it doesn't work at all. If it were, it'd be a simple text replacement before any interpretation and only the previously known characters would ever make it into the score file.

Reply:
@MarcSabatella: I'd really like to see what you had working